backend/tests/test_document_class.py

```diff
@@ -1,5 +1,33 @@
 from api.models import db, DocumentClass, Document
 
+import requests
```

Comment:
many of the document class tests were commented out because we didn't know how to test with auth - added header back in for testing and update tests accordingly

Reply:
this is already implemented in master